egs/iwslt18/mt1/run.sh

```diff
@@ -0,0 +1,276 @@
+#!/bin/bash
+
+# Copyright 2018 Kyoto University (Hirofumi Inaguma)
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+
+. ./path.sh
+. ./cmd.sh
+
+# general configuration
+backend=pytorch # chainer or pytorch
+stage=-1        # start from -1 if you need to start from data download
+stop_stage=100
+ngpu=1          # number of gpus ("0" uses cpu, otherwise use gpu)
+debugmode=1
+dumpdir=dump    # directory to dump full features
+N=0             # number of minibatches to be used (mainly for debugging). "0" uses all minibatches.
+verbose=0       # verbose option
+resume=         # Resume the training from snapshot
+seed=1          # seed to generate random number
+# feature configuration
```

Comment:
You can remove this feature configuration related things in the MT recipe.

Reply:
But there are a lot of ${do_delta} variable in the latter part...